Sources/WebViewProtocol.swift

```diff
@@ -16,7 +16,7 @@ public protocol WebViewProtocol: AnyObject {
     var navigationDelegate: WKNavigationDelegate? { get set }
     func add(_ scriptMessageHandler: WKScriptMessageHandler, name: String)
     func load(_ request: URLRequest) -> WKNavigation?
-    func evaluateJavaScript(_ javaScriptString: String, completionHandler: ((Any?, Error?) -> Void)?)
+    func evaluateJavaScript(_ javaScriptString: String, completionHandler: (@MainActor @Sendable (Any?, (any Error)?) -> Void)?)
```

Comment:
WDYTA making this change backward compatible?
```suggestion
    #if compiler(>=6)
        func evaluateJavaScript(_ javaScriptString: String, completionHandler: (@MainActor @Sendable (Any?, (any Error)?) -> Void)?)
    #else
        func evaluateJavaScript(_ javaScriptString: String, completionHandler: ((Any?, Error?) -> Void)?)
    #endif
```

Reply:
I wish we have a compiler < 6 around to test it with. But yes, that sounds great!

Reply:
I was able to test it using an older compiler, so we should be good to go 👍🏻

Reply:
moved to a separate PR: https://github.com/doconomy/dreams-ios-sdk/pull/14